### PR TITLE
Update maven in demos

### PR DESCRIPTION
**Pull request by cstamas**
Use latest Maven in Resolver Demos.

### CODE REVIEW

**Review by michael-o — APPROVED** [review]

File: maven-resolver-demos/maven-resolver-demo-snippets/src/main/java/org/apache/maven/resolver/examples/guice/DemoResolverModule.java [hunk 1]
```diff
@@ -39,6 +50,9 @@
 
 /**
  * Guice module for Demo Resolver snippets.
+ *
+ * Here, we assemble "complete" module by using {@link AetherModule} (see it's Javadoc) and adding bits from
```
Comment (michael-o) [inline maven-resolver-demos/maven-resolver-demo-snippets/src/main/java/org/apache/maven/resolver/examples/guice/DemoResolverModule.java:54]: No apostrophe

**Review by michael-o — CHANGES_REQUESTED** [review]
One nit and will test tomorrow.

**Comment by jira-importer** [comment]
Resolve #770

**Comment by jira-importer** [comment]
Resolve #770

**Pull request closed without merging**